### PR TITLE
Vet Clinic: Feature/groupby

### PR DESCRIPTION
**Pull request by mateo951**
- Added species column & new data inserted
- Add new queries

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,9 @@ insert into animals values (default, 'Agumon', '20200206', 1, true, 14.23);
 insert into animals values (default, 'Gabumon', '20181115', 2, true, 8.00);
 insert into animals values (default, 'Pikachu', '20210107', 1, false, 15.04);
 insert into animals values (default, 'Devimon', '20170512', 5, true, 11.00);
+insert into animals values (default, 'Charmander', '20200208', 0, false, 11.00);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's update weight_kg from 11.00 to -11.00 as given in the guidelines.

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,9 @@ insert into animals values (default, 'Agumon', '20200206', 1, true, 14.23);
...
+insert into animals values (default, 'Plantmon', '20221115', 2, true, 5.7);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's update weight_kg from 5.7 to -5.7 as given in the guidelines.

File: data.sql [hunk 3]
```diff
@@ -4,3 +4,9 @@ insert into animals values (default, 'Agumon', '20200206', 1, true, 14.23);
...
+insert into animals values (default, 'Squirtle', '19930402', 3, false, 12.13);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's update weight_kg from 12.13 to -12.13 as given in the guidelines.

File: data.sql [hunk 4]
```diff
@@ -4,3 +4,9 @@ insert into animals values (default, 'Agumon', '20200206', 1, true, 14.23);
...
+insert into animals values (default, 'Angemon', '20050612', 1, true, 45);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's update weight_kg from 45.00 to -45.00 as given in the guidelines.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: data.sql (hunk 1 above)
Comment (mateo951) [reply data.sql]: Done ✔

File: data.sql (hunk 2 above)
Comment (mateo951) [reply data.sql]: Done ✔

File: data.sql (hunk 3 above)
Comment (mateo951) [reply data.sql]: Done ✔

File: data.sql (hunk 4 above)
Comment (mateo951) [reply data.sql]: Done ✔

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @mateo951, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**